org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/AbstractLanguageServerTest.xtend

```diff
@@ -217,13 +219,26 @@ abstract class AbstractLanguageServerTest implements Endpoint {
 	}
 	
 	protected def InitializeResult initialize((InitializeParams)=>void initializer, boolean callInitialized) {
+		initialize(initializer, callInitialized, false)
+	}
+	
+	protected def InitializeResult initialize((InitializeParams)=>void initializer, boolean callInitialized, boolean useRootPath) {
 		val params = new InitializeParams => [
 			processId = 1
 			workspaceFolders = #[
 				new WorkspaceFolder(root.toURI.normalize.toUriString, '')
 			]
 		]
 		initializer?.apply(params)
```

Comment:
@szarnekow when I move the initializer after the useRootPath if, the tests fail with NullPointers. I will have to take a look into this.

Reply:
Did you figure why these failed when the order is swapped?

Reply:
I'm getting closer and closer. But I'm having some problems getting the tests to run locally. Is there any guidance/documentation on how to get them running locally.

Reply:
Which tests and what is the problem you are facing (maybe you have to select junit4 in the run dialog )

Reply:
The workspace was not setup correctly, but I don't know why. Everything was red and everything I tried didn't get it fixed. I now recreated the whole workspace with the eclipse installer and they are working again :)

Reply:
@szarnekow The reason for the NullPointerExceptions were some test classes where the WorkspaceFolder capability was not setup correctly.